user/doc.go

```diff
@@ -0,0 +1,48 @@
+/*
+Package user provides functionality related to NeoFS users.
+
+User identity is reflected in ID type. Each user has its own unique identifier
+within the same network.
+
+NeoFS user identification is compatible with Neo accounts:
+	import "crypto/ecdsa"
```

Comment:
what does that import mean/explain in the current doc? maybe add `keys` import instead?